spec/javascripts/health-care/components/personal-information/VeteranAddressSection.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import VeteranAddressSection from '../../../../../_health-care/_js/components/personal-information/VeteranAddressSection';
+
+describe('<VeteranAddressSection>', () => {
+  const nullAddress = [{
+    street: null,
+    city: null,
+    country: null,
+    state: null,
+    zipcode: null,
+  }];
+
+  it('Sanity check the component renders', () => {
+    const tree = SkinDeep.shallowRender(<VeteranAddressSection data={{ address: nullAddress }}/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.have.property('type', 'fieldset');
+  });
+
+  describe('Email confimration', () => {
+    it('does not include `error` prop when matches Email', () => {
+      const tree = SkinDeep.shallowRender(
+        <VeteranAddressSection data={{ address: nullAddress, email: 'mock@aol.com', emailConfirmation: 'mock@aol.com' }}/>);
+      const emailInputs = tree.everySubTree('Email');
+      expect(emailInputs).to.have.lengthOf(2);
+      expect(emailInputs[1].props.error).to.be.undefined;
+    });
+
+    it('does include `error` prop when matches Email', () => {
+      const tree = SkinDeep.shallowRender(
+        <VeteranAddressSection data={{ address: nullAddress, email: '', emailConfirmation: 'mock@aol.com' }}/>);
```

Comment:
This test is check against a function that already exists to check this so I don't know that we need both test cases. I am checking if the two props match and if they don't (whether blank or not equal) the `confirmEmail()` will catch it because it is looking for equality.